### PR TITLE
Update component style doc regarding relative URLs

### PR DESCRIPTION
**Pull request by dariobraun**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?
The current documentation does not match the mechanics of angular apps built with the CLI.

Issue Number: #22956

## What is the new behavior?
The documentation gives information in accordance with the up to date way of building angular apps - the CLI. This helps minimize confusion.
Explains how files are referenced when included in CLI built process.
Alternative: Only remove the following line:
`The link tag's href URL must be relative to the application root, not relative to the component file.`

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by dariobraun** [comment]
Is this still relevant? If so I'd appreciate a new review

File: aio/content/guide/component-styles.md [hunk 1]
```diff
@@ -216,11 +216,10 @@ You can also write `<link>` tags into the component's HTML template.
 
 <div class="alert is-critical">
 
-The link tag's `href` URL must be relative to the
-_**application root**_, not relative to the component file.
-
 When building with the CLI, be sure to include the linked style file among the assets to be copied to the server as described in the [CLI documentation](https://github.com/angular/angular-cli/wiki/stories-asset-configuration).
 
+Once included, angular will find the stylesheet, whether the link tag's href URL is relative to the application root or the component file.
+
```
Comment (brandonroberts) [inline aio/content/guide/component-styles.md]: I would update this to `Once included, the CLI will include the stylesheet...`

**Review by brandonroberts — CHANGES_REQUESTED** [review]

**Comment by brandonroberts** [comment]
@dariobraun I left you some feedback. Will you rebase on master, squash down to a single commit, and change the git commit message to `docs: update component styles doc regarding relative URL`.

Thanks

**Comment by dariobraun** [comment]
@brandonroberts Thanks for the feedback! I incorporated it :)

**Review by brandonroberts — APPROVED** [review]

**Pull request closed without merging**